flink-runtime-web/web-dashboard/src/app/services/logs-bundler.service.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { EMPTY } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { BASE_URL } from '../app.config';
+import {LogsBundlerStatus} from "../interfaces/logs-bundler";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class LogsBundlerService {
+    constructor(private httpClient: HttpClient) {}
+
+    /**
+     * trigger log bundling
+     */
+    triggerBundle() {
+        var res = this.httpClient.get(`${BASE_URL}/logbundler?action=trigger`).pipe(catchError(() => {
+            return EMPTY
```

Comment:
Mapping errors to `EMPTY` means this observable will emit nothing (and just complete) in a failure case. I'll have to read on to see if this is problematic. 

Edit from my future self: It means that if the API starts failing, the UI will simply not update anymore. Probably not what we'd want, so a nicer solution might be to map to a "fake" response, or let the error propagate and handle it at the caller site.

This is better than doing nothing at all, at least, since it prevents the error from clogging up the console. But it would leave the user in the dark. I'll leave it to you whether this is something we want to address or not given that this is probably the status quo of how things are done in Flink UI and thus this doesn't make it worse.